### ISSUE TITLE
Image generation suggestion

### ISSUE THREAD

**Issue by szhabolcs** [jonathanbossenger/devdadjokes#18]
Hello there @jonathanbossenger !
I really like the new changes, the page is getting better and better 😄 

One suggestion I'd make is to generate PNGs instead of JPGs, because the rounded corners get a black background, and if we export to a PNG they would have a transparent background afaik.
![DevDadJokes](https://user-images.githubusercontent.com/54114237/111107881-60a96180-8560-11eb-9586-e23903687ae8.jpeg)
(it's maybe hard to notice but it's there)

Another thing I'd like to also mention here, is that we should modify the DOM somehow before making the image, so the fake window would have that pattern background. I'm not sure how could we implement this in the nicest way, maybe we could add a parent div to the fake window, give it the pattern background, and then take the picture.

**Comment by jonathanbossenger** [comment]
Yeah, png is possible, mostly I was just playing around with the dom-to-image package this weekend -> https://github.com/tsayen/dom-to-image

I also had to move the icons from svg to png, and change how they get swapped out, due to that package not working with the svg's set as the `content` CSS attribute. 

(Now that I think about it, I might be able to use the svg's again, just as the img src instead of the content attribute)

I think it would be possible to add a patterned background, the code takes the entire `window` dom element into a JS variable and then converts that to an email, so I assume it would be possible to add some background to the dom variable.

I'll probably play with it some more this coming weekend.

**Comment by jonathanbossenger** [comment]
Partially fixed in #27

Adding the patterned background outstanding.
